src/Network/Receive/iRO.pm

```diff
@@ -33,6 +33,9 @@ sub new {
 		'0097' => ['private_message', 'v Z24 V Z*', [qw(len privMsgUser flag privMsg)]], # -1
 		'082D' => ['received_characters_info', 'x2 C5 x20', [qw(normal_slot premium_slot billing_slot producible_slot valid_slot)]],
 		'099D' => ['received_characters', 'x2 a*', [qw(charInfo)]],
+		'0A23' => ['achievement_list', 'v v V V v V V', [qw(len ach_count total_points rank current_rank_points next_rank_points)]], # -1
```

Comment:
This seems wrong. The number of values unpacked (7) doesn't match the number of keys (6).